src/components/profile/MemberTeamContainer.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import TeamCard from '../cards/TeamCard';
+
+class MemberTeamContainer extends Component {
+  render() {
+    const { memberData } = this.props;
+    const teams = memberData.members;
+    const teamsList = teams.map(member => {
+      return (
+        <TeamCard
+          key={member.id}
+          ownerProfileUrl={'/users/' + member.username}
```

Comment:
Add memberId to the `getMemberProfile.js` response and update this line
```suggestion
          ownerProfileUrl={`/members/${member.id}`}
```